target/linux/ramips/dts/mt7621_mikrotik_routerboard-760igs.dts

```diff
@@ -0,0 +1,85 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7621_mikrotik_routerboard-7xx.dtsi"
+
+/ {
+	compatible = "mikrotik,routerboard-760igs", "mediatek,mt7621-soc";
+	model = "MikroTik RouterBOARD 760iGS";
+
+	aliases {
+		led-boot = &led_pwr;
+		led-failsafe = &led_pwr;
+		led-running = &led_pwr;
+		led-upgrade = &led_pwr;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_pwr: pwr {
+			label = "routerboard-760igs:blue:pwr";
+			gpios = <&gpio 16 GPIO_ACTIVE_HIGH>;
+			default-state = "on";
+		};
+
+		sfp {
+			label = "routerboard-760igs:blue:sfp";
+			gpios = <&gpio 9 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	sfp1: sfp1 {
+		compatible = "sff,sfp";
+		i2c-bus = <&i2c>;
+		los-gpio = <&gpio 6 GPIO_ACTIVE_HIGH>;
+		mod-def0-gpio = <&gpio 8 GPIO_ACTIVE_LOW>;
+		tx-disable-gpio = <&gpio 10 GPIO_ACTIVE_HIGH>;
```

Comment:
Will change these nodes to use `-gpios`, as the SFP binding document specifies.
https://github.com/torvalds/linux/blob/d8b9faec54ae4bc2fff68bcd0befa93ace8256ce/Documentation/devicetree/bindings/net/sff%2Csfp.txt#L15
Functional either way.